### ISSUE TITLE
board-id [Mac-65CE76090165799A] and model [iMac 17,1] mismatch – check SMBIOS data

### ISSUE THREAD

**Issue by zorrobyte** [Piker-Alpha/ssdtPRGen.sh#248]
Trying to figure out why this is a mismatch as:
https://pikeralpha.wordpress.com/2015/10/14/os-x-10-11-1-beta-4-adds-skylake-support/

board-id: Mac-65CE76090165799A – Model: iMac17,1 (Retina 5K, 27-inch, Late 2015) / Core i7 4.0GHz

Optimally, could I safely ignore this error? If not, what should be the expected board-id?

**Comment by Piker-Alpha** [comment]
I don't know what arguments you used and what hardware you are using but we had two duplicates that may contribute to this issue. Please re-test with the latest update (v19.0).

Thank you for reporting this issue here!

**Comment by zorrobyte** [comment]
No worries!

This is a GA-Z170N-WIFI with a 6700K @ 4.8GHZ.

![screen shot 2016-08-03 at 7 15 57 pm](https://cloud.githubusercontent.com/assets/7257172/17385147/cb2c40c8-59ae-11e6-8b20-d7be144e886d.png)

This is the weird thing though (not related to ssdtPRGEN)
![screen shot 2016-08-03 at 7 19 40 pm](https://cloud.githubusercontent.com/assets/7257172/17385220/513e5d72-59af-11e6-83ab-c8667b096a19.png)

I even tried recreating my config.plist from scratch using clover and checking the generated plist file in Atom. All seems well, not sure why it's still showing as a Mac Pro 08

Regardless, frequency scaling and all seems to work just fine. I'll idle @ 800Mhz 16W and boost up to 4.8GHZ reliably (I enabled C2, C4 and C6 states). Never got sleep to work (even with proper DSDT, dumped from Clover boot and edited to compile) but I don't mind as I simply set it not to sleep.

Here is the full console log:
`Last login: Wed Aug  3 19:14:41 on ttys000
iMac:~ rossfisher$ curl -o ~/ssdtPRGen.sh https://raw.githubusercontent.com/Piker-Alpha/ssdtPRGen.sh/Beta/ssdtPRGen.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  147k  100  147k    0     0  85798      0  0:00:01  0:00:01 --:--:-- 85823
iMac:~ rossfisher$ chmod +x ~/ssdtPRGen.sh
iMac:~ rossfisher$ ~/ssdtPRGen.sh -f 4800

ssdtPRGen.sh v0.9  Copyright (c) 2011-2012 by † RevoGirl
             v6.6  Copyright (c) 2013 by † Jeroen

##              v19.0 Copyright (c) 2013-2016 by Pike R. Alpha

Bugs > https://github.com/Piker-Alpha/ssdtPRGen.sh/issues <

System information: Mac OS X 10.11.6 (15G31)
Brandstring: "Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz"

Override value: (-f) clock frequency, now using: 4800 MHz!

Notice: Downloading extractACPITables.zip ...
Notice: Downloading Sandy Bridge.cfg ...
Notice: Downloading Ivy Bridge.cfg ...
Notice: Downloading Haswell.cfg ...
Notice: Downloading Broadwell.cfg ...
Notice: Downloading Skylake.cfg ...
Version: models.cfg v160 / Sandy Bridge.cfg v0

Scope (_PR_) {222 bytes} with ACPI Processor declarations found in the DSDT (ACPI 1.0 compliant)
Notice: Downloading Restrictions.cfg ...
Generating ssdt.dsl for a 'iMac 17,1' with board-id [Mac-65CE76090165799A]
Skylake Core i7-6700K processor [0x506E3] setup [0x0705]
With a maximum TDP of 91 Watt, as specified by Intel
Number logical CPU's: 8 (Core Frequency: 4800 MHz)
Number of Turbo States: 0
Number of P-States: 35 (800-4200 MHz)
Injected C-States for CPU0 (C1,C3,C6,C7,C8,C9,C10)
Injected C-States for CPU1 (C1,C2,C3,C6,C7)
Warning: Model identifier (iMac 17,1) not found in..: /S_/L_/CoreServices/PlatformSupport.plist

Warning: 'cpu-type' may be set improperly (0x0705 instead of 0x0905)
Warning: board-id [Mac-65CE76090165799A] and model [iMac 17,1] mismatch – check SMBIOS data

Do you want to continue (y/n)? y
Notice: Downloading iasl.zip ...

Compiling: ssdt_pr.dsl
Intel ACPI Component Architecture
ASL Optimizing Compiler version 20140926-64 [Nov  6 2014]
Copyright (c) 2000 - 2014 Intel Corporation

ASL Input:     /Users/rossfisher/Library/ssdtPRGen/ssdt.dsl - 362 lines, 11334 bytes, 73 keywords
AML Output:    /Users/rossfisher/Library/ssdtPRGen/ssdt.aml - 2370 bytes, 28 named objects, 45 executable opcodes

Compilation complete. 0 Errors, 0 Warnings, 0 Remarks, 0 Optimizations

Do you want to copy /Users/rossfisher/Library/ssdtPRGen/ssdt.aml to /Extra/ssdt.aml? (y/n)? n
Do you want to open ssdt.dsl (y/n)? n
iMac:~ rossfisher$ `

**Comment by al3xtjames** [comment]
The model string in About This Mac is derived from the serial number; the last four digits represent the model identifier (Apple's servers return the model string). You can test this yourself: For example, for iMac13,2/DNMP: `curl -s "http://support-sp.apple.com/sp/index?page=cpuspec&cc=DNMP" | grep s.products | cut -d '"' -f 2 | cut -c 8-` will return `iMac (27-inch, Late 2012)`.

TL;DR: Change your serial number to one that matches iMac17,1.

**Comment by zorrobyte** [comment]
Heh, never knew that @theracermaster - thank you for the tip!

As I don't feel like regenerating some other variables to keep iMessage working, I'll just leave it be for the time being as frequency scaling and all seems to work properly. Cheers!

**Comment by Piker-Alpha** [comment]
Look at the warnings:

```
Warning: Model identifier (iMac 17,1) not found in..: /S/L/CoreServices/PlatformSupport.plist

Warning: board-id [Mac-65CE76090165799A] and model [iMac 17,1] mismatch – check SMBIOS data
```

Now note the spaces in the model identifier. That is the problem. And I couldn't tell that before, because you didn't copy/type the space character ;)
